ansible_collections/arista/avd/extensions/molecule/eos_designs_negative_unit_tests/inventory/host_vars/failure-devices-and-node-key-2.yml

```diff
@@ -0,0 +1,18 @@
+---
+fabric_name: FABRIC_FAILURE_DEVICES_AND_NODE_KEY_2
+
+type: l2leaf
+
+# Testing just assigning the profile is enough to trigger this.
+device_profile: test
+
+device_profiles:
+  - name: test
+
+l2leaf:
+  nodes:
+    - name: failure-devices-and-node-key-2
+
+expected_error_message: >-
+  Found the device 'failure-devices-and-node-key-2' under both 'l2leaf' and 'devices'.
```

Comment:
this error message may lead to people scratching their head given that they don't have `devices` defined

Reply:
hmm.. ok I will add yet another check for determine if it is devices or device_profile alone.